waspc/data/Generator/templates/server/src/jobs/core/pgBoss/pgBoss.js

```diff
@@ -1,9 +1,17 @@
 import PgBoss from 'pg-boss'
 import config from '../../../config.js'
 
-export const boss = new PgBoss({ connectionString: config.databaseUrl })
+// Add an escape hatch for advanced configuration of pg-boss.
+const pgBossNewOptions = process.env.PG_BOSS_NEW_OPTIONS || {}
+export const boss = new PgBoss({ connectionString: config.databaseUrl, ...pgBossNewOptions })
```

Comment:
I added this for emergency cases, like if they need to tweak the max number of db connections (or anything else here: https://github.com/timgit/pg-boss/blob/master/docs/readme.md#newoptions). I did not document it though, since I think 99.99% of the time it won't be needed, but just wanted to have in case the emergency case arose.

Thoughts on how we should handle and/or publicize these types of things?

Reply:
Ooof, you are correct on string. I do need to parse that 🤦🏻‍♂️ Will parse into object (better test it!) and document it. Ugh thanks

Reply:
Updated it to properly parse string into obj, which will overwrite our own settings if they so choose. I will add it to the docs later as well. 👍🏻